test/unit/privacy/checker_test.rb

```diff
@@ -34,21 +49,29 @@ class CheckerTest < Minitest::Test
       end
 
       test 'complains about private constant if enforcing for specific constants' do
```

Comment:
I guess I hadn't properly removed these tests so they didn't make sense before 😅 

Thanks for fixing!